HCIDOCS-651: Oracle Cloud C3 (#88897)
openshift/openshift-docs: modules/c3-assisted-installer-completing-installation-nodes.adoc

// Module included in the following assemblies:
//
// * installing/installing_oci/installing-c3-assisted-installer.adoc

:_mod-docs-content-type: PROCEDURE
[id="c3-ai-completing-installation-nodes_{context}"]
= Assigning node roles

> If the Terraform scripts completed successfully, twelve hosts are now listed for the cluster. Three control plane hosts and three compute hosts have the status *Disconnected*. Three control plane hosts and three compute hosts have the status *Insufficient*. 

Reviewer: ```suggestion
If the Terraform scripts completed successfully, twelve hosts are now listed for the cluster. Three control plane hosts and three compute hosts have the "Disconnected" status. Three control plane hosts and three compute hosts have the "Insufficient" status. 
```

.Procedure

. From the link:https://console.redhat.com/openshift/assisted-installer/clusters[{ai-full} web console], select the cluster and navigate to the *Host discovery* page. 
+

> If the Terraform scripts completed successfully, twelve hosts are listed on the page. Six hosts have the status *Disconnected* caused when you restarted the isntance on C3.

Reviewer 2: What do you mean twelve hosts ? we can install cluster with 3 master ot even 1... ? not sure about this context.

Author: It was in the Oracle document.

> . Delete the six hosts with a *Disconnected* status, by clicking the option button for the host and selecting *Remove host*. The status of the remaining hosts changes from *Insufficient* to *Ready*. This can take up to three minutes.

Reviewer: ```suggestion
. Delete the six hosts with a "Disconnected" status by clicking the option button for each host and selecting *Remove host*. The status of the remaining hosts changes from "Insufficient" to "Ready". This process can take up to three minutes.
```

> . From the *Role* column, assign the three nodes with a boot size of 1TB the *Control plane* role, and the three nodes with boot size 100GB the *Worker* role. 

Reviewer 2: assign the three nodes with a boot size of 1TB the *Control plane* ?  you mean here to boot disk size ,  why its 1TB ?  or 100GB , if it depends on  customer hardware it may be different

Author: It was in the Oracle document.

> . From the *Role* column, assign the three nodes with a boot size of 1.10TB the *Control plane* role, and the three nodes with boot size 100GB the *Worker* role. 

Reviewer: ```suggestion
. From the *Role* column, assign the *Control plane* role to the three nodes with a boot size of 1.10 TB. Assign the *Worker* role to the three nodes with boot size of 100 GB. 
```

> . Rename the hosts with a name shorter than 63 characters, by clicking the option button for the host and selecting *Change hostname*. Otherwise the cluster installation will fail. 

Reviewer: ```suggestion
. Rename any hosts with a name shorter than 63 characters by clicking the option button for the host and selecting *Change hostname*. Otherwise, the cluster installation will fail. 
```
